avocado/core/html.py

```diff
@@ -115,7 +115,7 @@ def _get_sysinfo(self, sysinfo_file):
         return sysinfo_contents
 
     def hostname(self):
-        return self._get_sysinfo('hostname')
+        return self._get_sysinfo('hostname').strip()
```

Comment:
It does, that's why I was asking and not demanding :smile:. OK so let's be sure and use it only for the hostname for now.